src/Appwrite/Utopia/Response/Model/ImportValidationError.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+
+class ImportValidationError extends Any
+{
+    /**
+     * Get Name
+     *
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'Migration Validiation Error';
+    }
+
+    /**
+     * Get Type
+     *
+     * @return string
+     */
+    public function getType(): string
+    {
+        return Response::MODEL_IMPORT_VALIDATION_ERROR;
+    }
+}
```

Comment:
Why do we need a new model here if it's not adding anything over the `Any` base class ? I think its better to reuse that